components/BaseLinePlotComponent.vue

```diff
@@ -234,16 +238,46 @@ export default class BaseLinePlotComponent extends Vue {
     return this.cumulatedSplitTimes[virtualCurrentSplitIndex] || [];
   }
 
+  get goldDate() {
+    const attempts = this.parsedSplits.Run.AttemptHistory.Attempt;
+    const dateTime = attempts.find(a => this.timesToPlot[this.gold.x]['@_id'] === a['@_id'])?.['@_ended'].split(" ");
+    if (dateTime) {
+      const date = dateTime[0].split("/");
+      return `${date[2]}-${date[0]}-${date[1]} ${dateTime[1]}`;
+    }
+  }
+
   get plot_data() {
+    const dateList = this.timesToPlot.map(t => {
```

Comment:
As a small optimization, maybe we could compute this only when plotting by date ? You could just assign `[]` if you want to avoid angering the type system when we don't need the value